app/controllers/task_controller.rb

```diff
@@ -30,12 +30,45 @@ def delete
   	redirect_to action: "index"
   end
 
+  def edit
+  	@edit_id = params[:edit_id]
```

Comment:
Since you're just using `@edit_id` on the line below to retrieve the task, and not using it in the view, it doesn't need to be an instance variable. It can just be a local variable.